ppr-ui/src/components/mhrRegistration/HomeLocation/HomeLocationType.vue

```diff
@@ -324,6 +330,9 @@ export default defineComponent({
     watch(() => localState.isLocationTypeValid, (val: boolean) => {
       setValidation(MhrSectVal.LOCATION_VALID, MhrCompVal.LOCATION_TYPE_VALID, val)
     })
+    watch(() => props.validate, async (val: boolean) => {
+      await validateForms()
+    })
```

Comment:
Im not sure we need this, don't these forms already validate correctly when navigating to review and back?

Reply:
No, this one wasn't validating properly due to the radio buttons. Another component that validates forms inside of radio buttons used this watcher, and it got these forms to validate properly as well.

Reply:
I'm not sure i follow the requirement. 
When i try this locally or in Dev, it validates without the watcher. 

![Screen Shot 2022-12-05 at 1 34 56 PM](https://user-images.githubusercontent.com/53542131/205747458-c03dfacd-726f-4bde-b03f-9374aa5a7bd1.png)

![Screen Shot 2022-12-05 at 1 35 06 PM](https://user-images.githubusercontent.com/53542131/205747487-cdae81b1-7e56-468d-aa5a-2da4086ffc0d.png)

![Screen Shot 2022-12-05 at 1 35 13 PM](https://user-images.githubusercontent.com/53542131/205747511-b5f0a491-eadc-4b65-bcb5-493f14ceb310.png)

Reply:
Is there a requirement different than whats displayed above. This is triggered by navigating to review and confirm step.

Reply:
Yes it was a bug Divya found... If you select a radio button but don't input anything before navigating to review, when you go back the text input fields wont validate

Reply:
Take a look at #14324

Reply:
Ok sounds good, thank you!